src/main/java/xogame/board_utils/Generator.java

```diff
@@ -0,0 +1,91 @@
+package xogame.board_utils;
+
+import java.util.Arrays;
+
+import xogame.models.Board;
+import xogame.utils.Result;
+
+public class Generator {
+    public static Result<Board> fill_nth_row_of_the_board_with_symbol(int board_size, int rowNumber, int symbol) {
```

Comment:
Here we speak of boards and filling their rows but what the method actually does is creating an empty board with a prefilled row. I don't think this is a good API since how are we going to modify an existing board if we want to /maybe we will never want to thou/?